packages/ui/react-ui/src/components/Buttons/IconButton.tsx

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React from 'react';
+
+import { Button, type ButtonProps } from './Button';
+import { useThemeContext } from '../../hooks';
+import { Icon, type IconProps } from '../Icon';
+
+type IconButtonProps = Omit<ButtonProps, 'children'> & Pick<IconProps, 'icon'> & { label?: string };
```

Comment:
```suggestion
type IconButtonProps = Omit<ButtonProps, 'children'> & Pick<IconProps, 'icon'> & { label: string };
```
Label must be required, it is user-hostile to exclude it. If a developer disagrees, they can provide an empty string.